src/electionguard/decrypt_with_shares.py

```diff
@@ -133,28 +133,19 @@ def decrypt_tally(
     if contests is None:
         return None
 
-    spoiled_ballots = decrypt_spoiled_ballots(
```

Comment:
I separated out the `spoiled_ballots` component in `PlaintextTally` since a Spoiled Ballot can be a `PlaintextTally`.

Long term, this means the CiphertextTally also needs to have the `spoiled` component removed as well.